schemas/context/account-marketo.schema.json

```diff
@@ -0,0 +1,184 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/account-marketo",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Account",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/account",
+    "https://ns.adobe.com/xdm/context/account-sf"
```

Comment:
https://ns.adobe.com/xdm/context/account-sf is not a class. This is intended to be an array of classes that this mixin can be added to

Reply:
The account-marketo MIXIN needs to extend the account-sf mixin. Can we just put the mixin in the intendtoExtend array ? It should automatically be added to the account class as well.

Reply:
Added the account-sf in the allOf section of account-marketo and also added the "meta:extends" in the account-marketo mixin. 
Removed the account-sf reference from the "meta:intendToExtend"